proxies/live/apiproxy/policies/OAuthV2.GenerateAccessToken.xml

```diff
@@ -10,6 +10,7 @@
     </SupportedGrantTypes>
     <GenerateResponse enabled="false"/>
     <Attributes>
+      <Attribute name="access_token" ref="externalAccessToken" display="true"/>
```

Comment:
could be better named to disambiguate what this is / where it's from. Also add to appropriate documentation (called 'Access token usage' or something similar in Confluence)

Reply:
Renamed to external_access_token